src/google/protobuf/compiler/python/generator.cc

```diff
@@ -32,7 +32,6 @@
 #include "absl/container/flat_hash_set.h"
 #include "absl/log/absl_check.h"
 #include "absl/log/absl_log.h"
-#include "absl/memory/memory.h"
```

Comment:
We use both absl::WrapUnique and absl::make_unique in a bunch of places, I'm surprised this builds...

Reply:
It builds